lib/igniter/project/application.ex

```diff
@@ -42,6 +42,52 @@ defmodule Igniter.Project.Application do
     end
   end
 
+  defp expand_app_name(zipper) do
+    with :error <- Common.expand_literal(zipper),
+         :error <- expand_attribute(zipper) do
+      :error
+    end
+  end
+
+  defp expand_attribute(zipper) do
+    with {:@, _, [{attr, _, nil}]} = attr_getter <- zipper.node,
+         %{} = getter_range <- Sourceror.get_range(attr_getter),
```

Comment:
I don't think that we can rely on this unfortunately. Igniter is predicated on the fact that other things may have written to this zipper before this function was called, and so may not have line number info.  i think we have to "move out to the defmodules scope" and then move up until we find the module attribute declaration.

Reply:
Ah, yes, you're right. If that's the case, I won't need the `Module.move_up_to_defmodule/1` helper. If it's alright with you, I'd prefer to remove it from this PR and I can submit a followup shortly after.